src/components/Meeting.tsx

```diff
@@ -189,8 +189,8 @@ export default function Meeting({
     if (meeting[`contact_${i}_phone` as keyof typeof Meeting])
```

Comment:
Not sure how often folks enter a phone without a name (`contact_1_name` doesn't exist but `contact_1_phone` does), could the group name default in the button label?